src/autoupdater.ts

```diff
@@ -305,6 +429,7 @@ export class AutoUpdater {
     const readyStateFilter = this.config.pullRequestReadyState();
     if (readyStateFilter !== 'all') {
       ghCore.info('Checking PR ready state');
+      ghCore.info(`PR ${JSON.stringify(pull)}`);
```

Comment:
Is this for debugging?

Reply:
Yes, it appear inside the github lambda execution pane.